documentation/index.html.erb

```diff
@@ -418,6 +418,11 @@ Expressions
       about it (say, when using jQuery).
     </p>
     <%= code_for('objects_reserved') %>
+    <p>
+      CoffeeScript has a shortcut for creating objects when you want the key
+      to be set with a variable of the same name.
+    </p>
+    <%= code_for('objects_shorthand') %>
```

Comment:
Ah, my local commit didn't even have this in the commit, as I had just done `git fetch origin pull/1808/head:docs-shorthand-object-notation`